media/js/src/GraphMapping.js

```diff
@@ -212,7 +214,9 @@ const convertGraph = function(json) {
         gA5Min: forceFloat(json.a5_min),
 
         gXAxisMax: forceFloat(json.x_axis_max),
+        gXAxisMin: forceFloat(json.x_axis_min),
```

Comment:
I don't see any x_axis_min/y_axis_min properties in the Django models - I don't think these code changes will really do anything. I think we agreed that these weren't necessary, as the starting axis value should always be 0.

Reply:
We did agree to that, but Tom said he wanted min values to be adjustable. I'll add them to the model and set 0 as the default.